src/styles/blocks/card.scss

```diff
@@ -0,0 +1,89 @@
+.card {
+  width: 200px;
+  display: flex;
+  flex-direction: column;
+
+  box-sizing: border-box;
```

Comment:
Move ` box-sizing: border-box;` into `*` seletor and remove in other places

Reply:
Updates have been done, I hope it will correct)